contrib/win32/win32compat/win32_usertoken_utils.c

```diff
@@ -319,13 +321,28 @@ get_user_token(char* user, int impersonation) {
 	}
 
 	if (wcscmp(user_utf16, L"sshd") == 0) {
-		if ((token = generate_sshd_virtual_token()) != 0)
+		/* not running as system, try generating sshd token as admin */
+		if (!am_system() && (token = generate_sshd_token_as_nonsystem()) != 0)
```

Comment:
Why not add a check to do this only in the debug mode?
People might misuse this  (running sshd as non-system) and we have to spend time in debugging..

Reply:
I don't think that is a misuse. Some people are looking for this exact use case.